pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -14,13 +14,14 @@ import (
 
 func TestBootstrappedConstraints(t *testing.T) {
 	// ordering of expectedConstraintNames is important, we check it against scc.ByPriority
+	// TODO: Fix/Check order, changed HostNS,HostNetwork
```

Comment:
why did this change?

Reply:
put there temporarily, I've restored the order